modules/ossm-enabling-jaeger.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/v2x/customizing-installation-ossm.adoc
+
+
+[id="ossm-enabling-tracing_{context}"]
+= Enabling Tracing
+
+You enable tracing by specifying a tracing type and a sampling rate.
+
+.Default `all-in-one` Jaeger parameters
+[source,yaml]
+----
+apiVersion: maistra.io/v2
+kind: ServiceMeshControlPlane
+metadata:
+  name: basic
+spec:
+  version: v2.0
+  tracing:
+    sampling: 100
+    type: Jaeger
+----
+
+Currently the only tracing type that is supported is `Jaeger` .   Jaeger is enabled by default.  To disable tracing, set `type` to `None`.
```

Comment:
You don't enable Jaeger the same way you enable Kiali or Grafana (enabled: true) because upstream Istio supports more than one type of tracing.  So first you have to enable tracing, then specify the type of tracer, in this case the only supported one is Jaeger.  I updated the heading to "Enabling and disabling tracing".   Since enabling and configuring Jaeger were in two separate places in the SMCP, I put the information under two separate headings.